.fixtures.yml

```diff
@@ -0,0 +1,3 @@
+fixtures:
+  repositories:
+    stdlib: "git://github.com/puppetlabs/puppetlabs-stdlib.git"
```

Comment:
Please add `puppetlabs-stdlib` in section `dependencies` of `metadata.json`

Reply:
Done